data/presets/baking_oven/bakehouse.json

```diff
@@ -0,0 +1,24 @@
+{
+    "icon": "Bakehouse",
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "tags": {
+        "amenity": "baking_oven",
+        "cuisine": "Bakehouse"
```

Comment:
where did you find the tag `cuisine=Bakehouse`? As far as I can see, this tag is not used in OSM.

Reply:
yes sorry didnt noticed it did the necessary changes